tests/run-make/native-link-modifier-verbatim-linker/rmake.rs

```diff
@@ -3,9 +3,6 @@
 // This test is the same as native-link-modifier-rustc, but without rlibs.
 // See https://github.com/rust-lang/rust/issues/99425
 
-//@ ignore-apple
-// Reason: linking fails due to the unusual ".ext" staticlib name.
```

Comment:
This ignore was added in https://github.com/rust-lang/rust/pull/101806, and only later rationalized when converting to rmake.